client/tests/integration/manuscript-manager-template-test.coffee

```diff
@@ -168,4 +171,4 @@ test 'click delete confirmation overlay submit button', ->
     equal find(".card-content").length, 0, "The card is gone"
   andThen ->
     search = { method: "DELETE", url: "/task_templates/1" }
-    ok _.findWhere(server.requests, search), "It sends DELETE request to the server"
+    ok _.findWhere(server.responses, search), "It sends DELETE request to the server"
```

Comment:
When I looked a the `server.requests`, I only saw four `GET` requests.  When I looked at the `server.responses`, I saw all of the stubbed out HTTP requests.  

Is there a better way to do this?